tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -463,6 +470,9 @@ private static async Task WriteDiagnosticLog(TracerManager instance)
                     writer.WritePropertyName("data_streams_enabled");
                     writer.WriteValue(instanceSettings.IsDataStreamsMonitoringEnabled);
 
+                    writer.WritePropertyName("span_sampling_rules");
```

Comment:
Also think about updating `ConfigurationTelemetryCollector` which allows us to record who uses what config